packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -1328,6 +1328,8 @@ class _SelectableFragment with Selectable, ChangeNotifier implements TextLayoutM
   TextPosition? _textSelectionStart;
   TextPosition? _textSelectionEnd;
 
+  bool? _selectableContainsOriginWord;
```

Comment:
why nullable? shouldn't this just be true or false?

Reply:
I don't recall why I made this nullable. I will make it non-nullable.